src/components/app/details/triggerView/workflow/Workflow.tsx

```diff
@@ -28,6 +29,9 @@ export class Workflow extends Component<WorkflowProps> {
     }
 
     renderSourceNode(node: NodeAttr) {
+        let appId = this.props.match.params.appId
```

Comment:
Please use const instead of let if the variable is not getting reassigned

Reply:
it's getting reassigned below so let should be used here